website/utils/url_utils.ts

```diff
@@ -955,8 +960,12 @@ export async function parseViewerUrlParams(urlSearchParams: URLSearchParams): Pr
 
   // Parse data sources (URL or dataset/id pair)
   if (params.url) {
-    const imageUrls = tryDecodeURLList(params.url) ?? decodeURL(params.url);
-    const firstUrl = Array.isArray(imageUrls) ? imageUrls[0] : imageUrls;
+    const sceneUrls = tryDecodeURLList(params.url, " ") ?? [params.url];
+    const sourceUrls = sceneUrls.map((scene) => tryDecodeURLList(scene) ?? [decodeURL(scene)]);
```

Comment:
is this supposed to give you every source url of every scene? I guess I don't know what tryDecodeUrlList can return .  I assume decodeURL returns a single string?